RF_PBMC.R

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env Rscript
+
+
+# Load libraries
+suppressPackageStartupMessages(library(optparse))
+suppressPackageStartupMessages(library(logging))
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(randomForest))
+suppressPackageStartupMessages(library(Seurat))
+
+ 
+# process_data function follows Seurat tutorial for processing PBMC data set made available by 10x Genomics
+# dataset https://s3-us-west-2.amazonaws.com/10x.files/samples/cell/pbmc3k/pbmc3k_filtered_gene_bc_matrices.tar.gz
+# Seurat tutorial https://satijalab.org/seurat/pbmc3k_tutorial.html
+process_data <- function(input.data){
+	dense.size <- object.size(x = as.matrix(x = input.data))
+	print(dense.size)
+	seuratObj <- CreateSeuratObject(raw.data = input.data, min.cells = 3, min.genes = 200, project = "10X_data")
+	print(seuratObj)
```

Comment:
min.cells and min.genes can be optional arguments.

Reply:
move print statements to logging using the logging class.

Reply:
Comment added indicating TODO for parameterization
Comment added settomg TODO for moving from print statements to logging